### DISCUSSION TITLE
Welcome to FritzCall Discussions!

### DISCUSSION THREAD

**Discussion by DrMichael** [DrMichael/FritzCall#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gerdseltmann-star** [comment]
Hallo Herr Dr Michael

Erstmal Danke das Sie das App Fritz Call weiter plegen 
bei egnima2  ist der Fritcall Monitor noch nicht vollständig es fehlt die Verbindung bei Glasfaser FritzBox 7590 Dsl geht die Anzeige 

Mit freundlichen Grüßen  Gerd Seltmann

**Comment by DrMichael** [comment]
Schon die https://github.com/DrMichael/FritzCall/releases/tag/rev1657 installiert?

**Comment by gerdseltmann-star** [comment]
Ja habe ich  Danke für die schnelle Antwort
![IMG_20250827_112347025](https://github.com/user-attachments/assets/d3c71873-3644-4b03-8535-843f5294e47b)

**Comment by DrMichael** [comment]
Dann bräuchte ich mal die /tmp/FritzCall_okGetInfo.json. (Vorher Log auf DEBUG stellen, neustarten.)

**Comment by gerdseltmann-star** [comment]
Debug finde ich im fritz app nicht

**Comment by DrMichael** [comment]
In der Konfiguration von FritzCall
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/41f5b50e-59a0-4850-aeab-894fb0b108f1" />

**Comment by gerdseltmann-star** [comment]
So Steht es drinn aber im Temp Verzeichnis ist nichts von Fritz drinn

**Comment by DrMichael** [comment]
Gar nichts?!? Mal neu gestartet? Was ist das für eine Box, wleche Firmware?

**Comment by gerdseltmann-star** [comment]
Ja neu start gemacht  DM920 letzes Image von neignima2  im tmp steht nur ci plus und nn demon

**Comment by DrMichael** [comment]
Sorry, ich kenne das System nicht. Steht denn etwas in /usr/lib/enigma2/python/Plugins/Extensions/FritzCall?

**Comment by gerdseltmann-star** [comment]


**Reply by DrMichael** [reply]
Sollte alles da sein. (Auch wenn ich nicht verstehe, wo die Zahlen da herkommen.)
Vielleicht mal im Forum der Firmware fragen...

**Reply by dhwz** [reply]
Das ist doch ein ganz normales DreamOS Image.
Vermutlich hat er debug doch nicht aktiviert.

**Reply by gerdseltmann-star** [reply]
im fritz app steht loglevel auf Debug

**Reply by dhwz** [reply]
Du sprichst schon von FritzCall unter Erweiterungen? weil du immer von einer Fritz App sprichst? Und du hast die Einstellungen mit OK gespeichert und danach einen GUI Neustart aus dem Hauptmenü ausgeführt? Das Plugin schreibt daraufhin mehrere debug Files nach /tmp

**Reply by gerdseltmann-star** [reply]
ja steht aber nichts in tmp immer mit grüner taste gespeichert und neu start

**Reply by dhwz** [reply]
Lösch dich bitte mal den kompletten Pluginordner und installiere es anschließend neu mit
https://github.com/DrMichael/FritzCall/releases/download/rev1657/enigma2-plugin-extensions-fritzcall_20250822-rev1657_all.deb

**Reply by gerdseltmann-star** [reply]
ich habe es deinstaliert im tmp war was 
[FritzCall_okGetInfo.json](https://github.com/user-attachments/files/22005734/FritzCall_okGetInfo.json)

**Reply by gerdseltmann-star** [reply]
ist neu instaliert aber die  Datenrate ist nicht zu sehen alles  beim alten

**Reply by dhwz** [reply]
Ja weil du die Verbindung ja auch mit einem externen Modem aufbaust, nicht mit der Fritzbox?
Das sieht FritzCall nicht vor und weiß daher auch nicht was es auslesen soll.
Funktionierte das schonmal mit einem älteren FritzOS vor 8.20?

Du hast FTTH Glasfaser aber nur 170MBit stimmen die Angaben?

**Reply by gerdseltmann-star** [reply]
![Uploading IMG_20250827_131127552.jpg…]()

**Reply by gerdseltmann-star** [reply]
wenn ich den alten Dsl Anschlus nutze zeigt mir  bei fritz alles an

**Reply by dhwz** [reply]
Bild wurde leider nicht richtig hochgeladen

**Reply by gerdseltmann-star** [reply]
dann wäre das Thema erledigt eine neue Fritz Box kommt erst wenn die alte vom Blitz erwischt wird dann gehts weiter ich nutze noch alte Isdn Modems was in den Neuen FritzBox nicht mehr vorhanden ist 
![Uploading IMG_20250827_131127552.jpg…]()

**Comment by gerdseltmann-star** [comment]
wie es ausieht wird Egnima 2 nicht mehr gepflegt  die Zahlen im verzeichnis sind Telefon Nr mit dem Skin hat es nichts zu tun 
![IMG_20250827_124403930](https://github.com/user-attachments/assets/262b0979-ef8d-47bf-bafc-a426ef858700)

**Reply by DrMichael** [reply]
Sorry, reine Neugier...
Die Glasfaser ist nicht an der FritzBox, richtig? Dann kann das Plugin auch nichts dazu anzeigen.
Aber, warum zeigt der Status eine externe IP an und "verbunden mit Telekom"?

**Reply by dhwz** [reply]
> Sorry, reine Neugier... Die Glasfaser ist nicht an der FritzBox, richtig? Dann kann das Plugin auch nichts dazu anzeigen. Aber, warum zeigt der Status eine externe IP an und "verbunden mit Telekom"?

Ich glaub er hat da ein Glasfasermodem am WAN-Port hängen das sich dann "einwählt", daher stehn da ein paar Infos, aber nicht alle und auch nicht an der von dir erwarteten Stelle.

**Reply by gerdseltmann-star** [reply]
Ja genau wie im Bild von Fritz Box  zu sehen mit Wan verbunden  denoch werden parameter zur Dm 920 übertragen

**Reply by gerdseltmann-star** [reply]
![Bildschirmfoto_27-8-2025_142547_192 168 178 1](https://github.com/user-attachments/assets/bf2109ec-9481-4a65-bc6f-9bfbc46e41e0)

**Comment by DrMichael** [comment]
Interessant.

**Reply by gerdseltmann-star** [reply]
Genau Vielleicht lässt sich was draus machen euch nochmals Danke werde abundzu schauen ob was neues gibt Euch noch eine erfolgreiche Woche.

**Reply by gerdseltmann-star** [reply]
Moin vieleicht nutzt das noch im App müsste der Punkt DSl mehreren zugewiesen werden wie Viber Glasfaser und WAN es beschränkt sich eh nur um die Punkte 
![Bildschirmfoto_28-8-2025_9850_192 168 178 1](https://github.com/user-attachments/assets/be25ad5d-3a00-45c7-af82-c04fb3b3f329)

**Comment by DrMichael** [comment]
Ah, ich hatte mir das dann mal auf einer meiner FBs angeschaut, die hinter einem Glasfasermodem hängen. Dort wird "bis zu 1 Gbit/s" angezeigt, was ja aus Sicht der FB durchaus richtig ist. Aber eben nicht die Geschwindigkeit des Internetanschlusses angibt.
Daher möchte ich das nicht anzeigen lassen.
